Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json

```diff
@@ -5,5 +5,17 @@
     "en": "Binary sensor"
   },
   "extends": "dtmi:digitaltwins:rec_3_3:device:StateSensor;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "lastValue",
+      "displayName": {
```

Comment:
For consistency with the rest of the ontology and the underlying OWL models, a description should probably be added, see https://github.com/Azure/opendigitaltwins-building/blob/master/Ontology/Capability/Sensor/QuantitySensor/AreaSensor.json for example.

Reply:
Will add description as well